packages/filesystem/src/node/filesystem-backend-module.ts

```diff
@@ -36,18 +34,13 @@ export function bindFileSystem(bind: interfaces.Bind, props?: {
 }
 
 export function bindFileSystemWatcherServer(bind: interfaces.Bind): void {
-    if (cluster.isMaster) {
-        bind(FileSystemWatcherServer).toDynamicValue(ctx => {
-            const logger = ctx.container.get<ILogger>(ILogger);
-            return new NsfwFileSystemWatcherServer({
-                info: (message, ...args) => logger.info(message, ...args),
-                error: (message, ...args) => logger.error(message, ...args)
-            });
+    bind(FileSystemWatcherServer).toDynamicValue(ctx => {
+        const logger = ctx.container.get<ILogger>(ILogger);
+        return new NsfwFileSystemWatcherServer({
+            info: (message, ...args) => logger.info(message, ...args),
+            error: (message, ...args) => logger.error(message, ...args)
         });
-    } else {
-        bind(FileSystemWatcherServerClient).toSelf();
```

Comment:
it's prod implementation.

Reply:
I'm not sure what you mean. Surely the difference here is that one way is using cluster mode and one doesn't, they should both be `production`

Reply:
@akosyakov Can you confirm what you want here? Specifically, which section should be used in `--no-cluster` (single threaded mode) and which is `normal` mode.

Reply:
`normal` mode is `else` branch